.github/workflows/build_and_deploy.yml

```diff
@@ -473,6 +473,31 @@ jobs:
           VERCEL_API_TOKEN: ${{ secrets.VERCEL_API_TOKEN }}
           DEPLOY_ENVIRONMENT: production
 
+  deployTurbopackDocs:
+    name: Deploy Turbopack docs
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 25
+      - name: Install Vercel CLI
+        run: npm i -g vercel@latest
+      - name: Install Rust
+        uses: ./.github/actions/setup-rust
+      - name: Deploy preview docs
+        if: ${{ needs.build.outputs.isRelease != 'true' }}
```

Comment:
this could just be a ternary on `DEPLOY_ENVIRONMENT`

Reply:
Could be, but same as the script itself this mimics pattern setup in other deploy work - making it consistent so later easy to find & replace as needed.

Reply:
just because the existing code is bad doesn't mean the new code has to be as well

Reply:
is it bad though? I honestly don't think it's that bad, especially `isRelease` is universal flag for all of the existing next.js CI's workflow to work with release. It might be different, but I don't see to try other pattern over established pattern.